frontend-wasm/src/module/mod.rs

```diff
@@ -7,19 +7,22 @@ use std::{borrow::Cow, collections::BTreeMap, ops::Range};
 
 use indexmap::IndexMap;
 use miden_diagnostics::DiagnosticsHandler;
-use miden_hir::cranelift_entity::{packed_option::ReservedValue, EntityRef, PrimaryMap};
+use miden_hir::{
+    cranelift_entity::{packed_option::ReservedValue, EntityRef, PrimaryMap},
+    Ident, Symbol,
+};
 use rustc_hash::FxHashMap;
 
 use self::types::*;
 use crate::{component::SignatureIndex, error::WasmResult, unsupported_diag};
 
 pub mod build_ir;
-pub mod func_env;
 pub mod func_translation_state;
 pub mod func_translator;
 pub mod function_builder_ext;
 pub mod instance;
 pub mod module_env;
+pub mod module_tratnslation_state;
```

Comment:
*Typo:* A sneaky one! Almost didn't notice it haha

Reply:
Oh, indeed, a sneaky one! Thanks! Fixed.